.github/workflows/maestro_release.yaml

```diff
@@ -80,9 +79,56 @@ jobs:
         with:
           name: maestro-dist
           path: dist
+          
+      - uses: actions/checkout@v4
+      
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+
+      - name: Install uv and activate the environment
+        uses: astral-sh/setup-uv@v6
+        with:
+          activate-environment: true
+
+      - name: Install dependencies
+        run: |
+          uv pip install -e .[crewai]
+
+      - name: Get release name
+        id: get_name
+        run: python tools/get_release_name.py
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_REPOSITORY: ${{ github.repository }}
```

Comment:
IIUC we could move this to the build step and have that step output the value for later use below

Reply:
fyi https://docs.github.com/en/actions/how-tos/writing-workflows/choosing-what-your-workflow-does/passing-information-between-jobs